src/shape/bounding-sphere.js

```diff
@@ -99,10 +100,28 @@ pc.extend(pc, function () {
                 point.copy(ray.direction).scale(t).add(ray.origin);
 
             return true;
+        },
+
+        /**
+         * @function
+         * @name pc.BoundingSphere#intersectsBoundingSphere
+         * @description Test if a Bounding Sphere is inside this Bounding Sphere.
```

Comment:
Intersect !== inside.
Consider rephrasing this to make it clear what *intersect* is.